src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+body {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.card {
+  display: flexbox;
+  box-sizing: border-box;
+  width: $card-size;
+  border: 1px solid #f3f3f3;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)